### PR TITLE
i18n sync hourofcode.com down and out 08/27 - Spanish Only

### PR DESCRIPTION
**Pull request by Hamms**
Doing a Spanish-only sync in prep for LAHOC

### CODE REVIEW

File: i18n/locales/es-MX/hourofcode/how-to/afterschool.md [hunk 1]
```diff
@@ -5,121 +5,81 @@ nav: how_to_nav
...
+
+* * *
 
 ## 1. Mira este vídeo explicativo <iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen mark="crwd-mark"></iframe> 
 
-## 2. Elige un tutorial
+## 2. Selecciona un tutorial
 
-Nosotros proveemos una variedad de [entretenidos tutoriales de una hora ](%= resolve_url('/learn') %) para participantes de todas las edades, creado por una variedad de socios. [!Revísalos aquí!](%= resolve_url('/learn') %)
+We provide a variety of [fun, hour-long tutorials](%= resolve_url('/learn') %) for participants all ages, created by a variety of partners. [!Revísalos aquí!](%= resolve_url('/learn') %)
```
Comment (tanyaparker) [inline i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: How come we lost these translations? Could the English string have changed a bit? But I thought we fallback in the Hour of Code project?

Comment (Hamms) [reply i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: it's a property of automatic string detection in markdown in crowdin; significant enough changes in the file can actually result in the strings being identified as entirely new strings. So crowdin doesn't detect this as a string for which the english source changed, but actually as an entirely new string

Comment (Hamms) [reply i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: You can see the translation history for this specific string here: https://crowdin.com/project/hour-of-code/activity_stream?translation_id=169501&lang=72

Comment (tanyaparker) [reply i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: Ah, ok. Would it be ok if I go in and re-apply some of these translations that I think are still good? (I know some Spanish). Would you mind re-syncing after I do this? Or do you think it's not worth it?

Comment (Hamms) [reply i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: That seems pretty reasonable! I'd check with Suky to make sure you wouldn't be duplicating work the partners are already planning to do, but I think something like that would totally be worth it, and I'd be more than happy to re-sync

Comment (tanyaparker) [reply i18n/locales/es-MX/hourofcode/how-to/afterschool.md:18]: Ok checked with Suky. I won't be duplicating any work. I just went through the es-MX Hour of Code project. All set for a re-sync!

**Comment by Hamms** [comment]
closing in favor of https://github.com/code-dot-org/code-dot-org/pull/24662

**Pull request closed without merging**